<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>searching_page</class>
 <widget class="QMainWindow" name="searching_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Client">
     <attribute name="title">
      <string>客户订单</string>
     </attribute>
     <widget class="QListWidget" name="Client_list">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>49</y>
        <width>251</width>
        <height>551</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="search_edit">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="title">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>请输入查询内容</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_1">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>20</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>请选择</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>订单号</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>客户名称</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>20</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
     <widget class="QLabel" name="order_client_id">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>100</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>订单号：</string>
      </property>
     </widget>
     <widget class="QLabel" name="client_name">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>150</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>客户名：</string>
      </property>
     </widget>
     <widget class="QLabel" name="staff_name">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>210</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>跟进员工：</string>
      </property>
     </widget>
     <widget class="QLabel" name="shipment_name">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>270</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>货运公司：</string>
      </property>
     </widget>
     <widget class="QLabel" name="product_ID">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>100</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>商品号：</string>
      </property>
     </widget>
     <widget class="QLabel" name="product_type">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>150</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>商品类型：</string>
      </property>
     </widget>
     <widget class="QLabel" name="product_number">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>210</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>数量：</string>
      </property>
     </widget>
     <widget class="QLabel" name="finish_date">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>270</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>交货日期：</string>
      </property>
     </widget>
     <widget class="QLabel" name="departure">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>330</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>发出地：</string>
      </property>
     </widget>
     <widget class="QLabel" name="destination">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>330</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>收货地：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="order_client_ID_edit">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>100</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="product_ID_edit">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>100</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="client_name_edit">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>150</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="staff_name_edit">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>210</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="shipment_name_edit">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>270</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="departure_edit">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>330</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="product_type_edit">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>150</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="product_number_edit">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>210</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="finish_date_edit">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>270</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="destination_edit">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>330</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="way">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>330</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>货运方式：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="way_edit">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>330</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>400</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>查看货运</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>400</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>查看客户</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>470</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>修改</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>470</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
     <widget class="QPushButton" name="new_button">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>20</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>新建订单</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Factory">
     <attribute name="title">
      <string>工厂订单</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="Product">
     <attribute name="title">
      <string>商品信息</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
